release/models/acl/openconfig-packet-match-types.yang

```diff
@@ -371,4 +371,28 @@ module openconfig-packet-match-types {
       decimal notation, or using a type defined by the
       ETHERTYPE identity";
   }
+
+  typedef ipv4-prefix-mask-type {
+    type union {
+      type uint8 {
+        range 0..32;
+      }
+      type oc-inet:ipv4-address;
+    }
+    description
+      "The ipv4-prefix mask to be used in acl-entries. Can be in CIDR
+      formmat or wildcard bits.";
```

Comment:
It's not actually clear to me what this `typedef` means -- is it solely the mask?